src/components/navigation/Tabs.js

```diff
@@ -69,29 +84,31 @@ const Li = styled.li`
       @media (min-width: ${SCREEN_SM_MIN}) {
         position:relative;
         text-align:center;
-        a::after {
-            height: 3px;
-            display: block;
-            width: 100%;
-            background: ${reactBlue()};
-            content: '';
-            position: absolute;
-            bottom:-10px;
-            left:0;
-        }
       }
       @media (max-width: ${SCREEN_XS_MAX}) {
         border: 1px solid ${reactBlue()}
       }
+      background: ${selectTypeColor(props.name)}
     `
       : ''};
+   }
+`
+const A = styled.a`
+  ${props => 
+    `border-bottom: 3px solid ${selectTypeColor(props.name)}`  
+  };
+  ${props => {
+    if (props.isActive && (props.name === PART_TIME || props.name === REACT_BOOTCAMP)) {
+      return `color: white !important`
```

Comment:
do we really need the `important!` ??

Reply:
Unfortunately we do need it as when it's used on the landing page cards it gets overwritten if not due to the way we style <a> in that section. On the curriculum page tho it does work without.